runbooks/workflow/creating-maintaining-littlepay-data-syncs.md

```diff
@@ -0,0 +1,51 @@
+# Creating and Maintaining Littlepay Data Syncs
+
+> Some of this is taken from the provided Littlepay documentation, with Cal-ITP specific content added.
+
+We access Littlepay data via AWS S3 file storage, and use Airflow-orchestrated sync tasks to bring that data into Cal-ITP's GCS buckets and then into our data warehouse.
+
+## Setting Up a New Littlepay Sync
+
+### Prerequisites
+
+In order to access source data from Littlepay, you will need a vendor-provided AWS access key for the agency you're setting up. Examples of access keys for existing Littlepay data syncs can be found in [Google Cloud Secret Manager](https://console.cloud.google.com/security/secret-manager?project=cal-itp-data-infra).
```

Comment:
How are these keys usually provided? By email from Littlepay or some other way?

Reply:
Resolved in [0049f24](https://github.com/cal-itp/data-infra/pull/3019/commits/0049f24032f29d2a243d2db04fe472dcd4067ae7)